### PR TITLE
Fix for Jira Issues (EMM-953 , EMM-839)

### PR DESCRIPTION
**Pull request by Kamidu**

### CODE REVIEW

File: modules/apps/jaggery/mdm/src/mdm/units/device-listing/device-listing.hbs [hunk 1]
```diff
@@ -40,8 +40,16 @@
                 </th>
             </tr>
             </thead>
+            <div id="loading-content" class="col-centered">
+                <br>
+                <br>
+                <i class="fw fw-settings fw-spin fw-2x"></i>
+                &nbsp;&nbsp;&nbsp;
+                Loading policy creation wizard . . .
```
Comment (kasundananjaya) [inline modules/apps/jaggery/mdm/src/mdm/units/device-listing/device-listing.hbs]: Change this to load device list instead of policy creation wizard.

Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/units/device-listing/device-listing.hbs]: Change has been made

File: modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ var backendServiceInvoker = function () {
                 if (xmlHttpRequest.responseText != null) {
                     return successCallback(parse(xmlHttpRequest.responseText));
                 } else {
-                    return successCallback(null);
+                    return successCallback({"statusCode":200,"messageFromServer":"Operation Completed"});
```
Comment (kasundananjaya) [inline modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js]: Fix formatting issues.

Comment (kasundananjaya) [inline modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js]: Fix formatting issues.

File: modules/apps/jaggery/mdm/src/mdm/units/user-update/public/js/bottomJs.js [hunk 3]
```diff
@@ -24,6 +24,10 @@ $(document).ready(function () {
     $("select.select2[multiple=multiple]").select2({
         tags : true
     });
+    var roleList = $("#roles").attr("selectedVals").trim().replace(/ /g,"");
```
Comment (kasundananjaya) [inline modules/apps/jaggery/mdm/src/mdm/units/user-update/public/js/bottomJs.js]: Fix formatting issues.

File: modules/apps/jaggery/mdm/src/mdm/units/user-update/public/js/bottomJs.js [hunk 4]
```diff
@@ -24,6 +24,10 @@ $(document).ready(function () {
...
+    roleList = roleList.replace(/(\r\n|\n|\r)/gm,"");
```
Comment (kasundananjaya) [inline modules/apps/jaggery/mdm/src/mdm/units/user-update/public/js/bottomJs.js]: Fix formatting issues.

File: modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js (hunk 2 above)
Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js]: These formatting issues has been fixed in a recent PR

File: modules/apps/jaggery/mdm/src/mdm/units/user-update/public/js/bottomJs.js (hunk 3 above)
Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/units/user-update/public/js/bottomJs.js]: Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158

File: modules/apps/jaggery/mdm/src/mdm/units/user-update/public/js/bottomJs.js (hunk 4 above)
Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/units/user-update/public/js/bottomJs.js]: Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158

**Pull request merged**